WORKSPACE

```diff
@@ -117,44 +69,17 @@ com_github_grpc_grpc_bazel_grpc_deps()
 load("@stackb_rules_proto//java:deps.bzl", "java_grpc_compile")
 java_grpc_compile()
 
-# Load GRPC Python dependencies
-load("@stackb_rules_proto//python:deps.bzl", "python_grpc_compile")
-python_grpc_compile()
-
-# Load GRPC Node.js dependencies
-load("@stackb_rules_proto//node:deps.bzl", "node_grpc_compile")
-node_grpc_compile()
-
 
 ########################################
 #     Load Deployment Dependencies     #
 ########################################
```

Comment:
We should also rename this to `Load Distribution Dependencies`

Reply:
Good catch, fixed in https://github.com/graknlabs/grakn/pull/4906/commits/6d68ce57ce6cb9b656b2607fd2225617f5c83639